Aardvark.podspec

```diff
@@ -7,7 +7,7 @@ Pod::Spec.new do |s|
   s.authors  = 'Square'
   s.source   = { :git => 'https://github.com/square/Aardvark.git', :tag => s.version }
   s.swift_version = '4.0'
```

Comment:
Since this PR updates the Swift version in `Aardvark.xcodeproj`, I think we also need to bump the Swift version in the podspecs.

Reply:
Fixed in ecab5f1